master/buildbot/test/unit/test_secret_in_vault.py

```diff
@@ -110,23 +117,30 @@ def setUp(self):
 
     @defer.inlineCallbacks
     def testGetValue(self):
-        self._http.expect(method='get', ep='/v1/secret/data/value', params=None,
+        self._http.expect(method='get', ep='/v1/secret/data/', params=None,
                           data=None, json=None, code=200,
                           content_json={"data": {"data": {"value": "value1"}}})
         value = yield self.srvcVault.get("value")
         self.assertEqual(value, "value1")
 
+    @defer.inlineCallbacks
+    def testGetAnyKey(self):
+        self._http.expect(method='get', ep='/v1/secret/data/', params=None,
+                          data=None, json=None, code=200,
+                          content_json={"data": {"data": {"any_key": "value1"}}})
+        value = yield self.srvcVault.get("any_key")
+        self.assertEqual(value, "value1")
```

Comment:
How would I get this value from a Renderable object? This is not very clear for me, we need a test for it, and then documentation

Reply:
Fixed now.